hack/ensure-blockcsi-cryaml.sh

```diff
@@ -21,9 +21,9 @@ set -e
 source hack/common.sh
 
 CSI_CR_PATH="config/manager/${BLOCK_CSI_CR_FILE}"
-CSI_CR_URL="https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/${BLOCK_CSI_RELEASE}/deploy/crds/${BLOCK_CSI_CR_FILE}"
+CSI_CR_URL="https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/${BLOCK_CSI_RELEASE}/config/samples/csi.ibm.com_v1_ibmblockcsi_cr.yaml"
 
-if [ ! -f "${CSI_CR_PATH}" ] || [[ -f "${CSI_CR_PATH}" && "$(grep release "${CSI_CR_PATH}" | awk -F ': ' '{print $2}')" != ${BLOCK_CSI_RELEASE} ]]; then
+if curl --head --silent --fail ${CSI_CR_URL} 2> /dev/null; then
         echo "Downloading the IBM Block CSI CR file..."
         curl -JL "${CSI_CR_URL}" -o "${CSI_CR_PATH}"
```

Comment:
Also, please copy the CR file into config/samples

For some reason this file is duplicated and until we figure out why, we would like to maintain them both